doc/requirements.txt

```diff
@@ -0,0 +1,8 @@
+pyside2
+numpy
+scipy
+h5py
+matplotlib
```

Comment:
Just out of curiosity why is `matplotlib` needed here?

Reply:
MatplotlibWidget

Reply:
We could [mock it](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_mock_imports)

Reply:
Oh right, forgot about that, I'm good without mocking unless you think we should.

Reply:
I went ahead and trimmed it down to what's necessary, if only to speed up builds.